### PR TITLE
Replace var with const and let  

### PR DESCRIPTION
**Pull request by cameronaaron**

### CODE REVIEW

**Comment by alalamav** [comment]
I think it should be fine to use ES6 features here, since the node/JS runtime is provided by Electron and not the client browser (no transpilation required either).

File: src/server_manager/electron_app/digitalocean_oauth.ts [hunk 1]
```diff
@@ -147,18 +146,18 @@ export function runOauth(): OauthSession {
               <script>
                   // We can't use URLSearchParams in IE :-(
                   function splitParams(paramsStr) {
-                    var params = {};
-                    var kvs = paramsStr.split("&");
-                    for (var i in kvs) {
+                    const params = {};
```
Comment (bemasc) [inline src/server_manager/electron_app/digitalocean_oauth.ts]: This code has to run in arbitrary old web browsers so it has to use `var`.

Comment (fortuna) [reply src/server_manager/electron_app/digitalocean_oauth.ts]: +1 Please revert this file.

File: src/server_manager/ui_components/outline-region-picker-step.html [hunk 2]
```diff
@@ -41,30 +41,27 @@
       }
 
       input[type="radio"]:checked+label.city-button:hover {
-        background: #FFFFFF;
         box-shadow: 0 4px 5px 0 rgba(0,0,0,0.14), 0 1px 10px 0 rgba(0,0,0,0.12), 0 2px 4px 0 rgba(0,0,0,0.20);
         border-radius: 3px;
         /* from #00bfa5 */
-        background-color: rgba(0, 191, 165, 0.12);
+          background: rgba(0, 191, 165, 0.12);
```
Comment (fortuna) [inline src/server_manager/ui_components/outline-region-picker-step.html]: Revert?

File: src/server_manager/ui_components/outline-server-creator.html [hunk 3]
```diff
@@ -145,11 +145,8 @@
 
       hr {
         width: 46px;
-        border-width: 0px 0px 1px 0px;
-        border-style: solid;
-        border-color: rgba(0, 0, 0, 0.24);
-        margin: 0px 24px 2px 24px;
-        border: 1px, 1px, 1px, 1px;
+        margin: 0 24px 2px 24px;
+          border: 1px 1px 1px 1px solid rgba(0, 0, 0, 0.24);
```
Comment (fortuna) [inline src/server_manager/ui_components/outline-server-creator.html]: Alignment

File: src/server_manager/ui_components/outline-server-progress-step.html [hunk 4]
```diff
@@ -41,9 +41,8 @@
 
       h3 {
         font-size: 18px;
-        margin-bottom: 0px;
         color: rgba(0,0,0,0.54);
-        margin: 0px 0px 0px 0px;
+          margin: 0;
```
Comment (fortuna) [inline src/server_manager/ui_components/outline-server-progress-step.html]: Alignment

File: src/server_manager/ui_components/outline-share-dialog.html [hunk 5]
```diff
@@ -139,8 +139,8 @@ <h3>Share access</h3>
         this.$.dialog.open();
       },
       copyClicked: function() {
-        var dt = new this.clipboard.DT();
-        dt.setData("text/plain", this.$.selectableText.innerText);
+          const dt = new this.clipboard.DT();
```
Comment (fortuna) [inline src/server_manager/ui_components/outline-share-dialog.html]: Alignment

File: scripts/do_action.sh [hunk 6]
```diff
@@ -29,9 +29,11 @@ function do_action() {
...
   shift
-  $ROOT_DIR/src/${action}_action.sh "$@"
+  ${ROOT_DIR}/src/${action}_action.sh "$@"
   echo -e "$STYLE_BOLD_WHITE[Done $action]$STYLE_RESET"
 }
 export -f do_action
 
 do_action "$@"
+
+
```
Comment (fortuna) [inline scripts/do_action.sh]: Remove extra line.

File: scripts/do_action.sh [hunk 7]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (fortuna) [inline scripts/do_action.sh]: Why this change?

**Comment by trevj** [comment]
@cameronaaron Apologies for the slow updates on this. Given the volume of changes since you originally submitted this for review, I'd suggest you:
* create a new 
* use `const` wherever possible in `*.ts` files (except in `digitalocean_oauth.ts` as noted above)

How does that sound?

**Comment by trevj** [comment]
@cameronaaron I'm going to close just because I don't like to keep PRs hanging around too long - but if find time to tackle this again, please do and open another PR we'd love to have the contribution.
